test/test_CellArray.jl

```diff
@@ -53,30 +62,34 @@ mutable struct MyMutableFieldArray{T} <: FieldArray{Tuple{2}, T, 1}
     yxxx::T
 end
 
+const TEST_PRECISIONS = [Float32, Float64]
```

Comment:
@albert-de-montserrat could you integrate the changes you did in your PR that didnt fully work unfortunately in the tests concerning the float type?